projects/packages/calypsoify/.phan/baseline.php

```diff
@@ -9,12 +9,11 @@
  */
 return [
     // # Issue statistics:
-    // PhanTypeMismatchArgumentProbablyReal : 1 occurrence
     // PhanTypeMismatchPropertyDefault : 1 occurrence
 
     // Currently, file_suppressions and directory_suppressions are the only supported suppressions
     'file_suppressions' => [
-        'src/class-jetpack-calypsoify.php' => ['PhanTypeMismatchArgumentProbablyReal', 'PhanTypeMismatchPropertyDefault'],
+        'src/class-jetpack-calypsoify.php' => ['PhanTypeMismatchPropertyDefault'],
```

Comment:
I believe we can get rid of this entire file by fixing the issue in `class-jetpack-calypsoify.php` line 23 by replacing with `@var object` with `@var self|false`

Reply:
Nice, thanks!